src/main/js/controls/Bubble/helpers/helpers.js

```diff
@@ -266,13 +265,13 @@ const getAllCircleNodes = () => {
  * @param {Array} nodes - List of d3.selection objects that are circles form bubble graph
  * @returns {undefined} - returns nothing
  */
-const enforceBubbleBlur = (nodes) =>
-    nodes.forEach((ele) => {
-        const node = d3.select(ele);
-        if (node.attr("aria-selected") === "false") {
-            node.classed(styles.bubbleBlur, true);
-        }
+const enforceBubbleBlur = (nodes) => {
+    nodes.forEach((bubbleNode) => {
+        d3.select(bubbleNode.firstChild)
+            .attr("fill-opacity", "0.1")
```

Comment:
You might want to create constants for this similar to `constants.DEFAULT_BUBBLE_OPACITY`

Reply:
Ok, sure I will create constants for this too.